### PR TITLE
Run steam only on x86_64

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/76870
Verification (scheduling only):
    * *x86_64*: http://kazhua.qa.suse.de/tests/124
    * *ppc64le*: http://kazhua.qa.suse.de/tests/125

### CODE REVIEW

**Comment by foursixnine** [comment]
@rwx788 You might want to look at the changes...

File: lib/scheduler.pm [hunk 1]
```diff
@@ -51,17 +51,29 @@ sub parse_schedule {
     my ($schedule) = shift;
     my @scheduled;
     for my $module (@{$schedule->{schedule}}) {
-        push(@scheduled, $module) && next unless ($module =~ s/\{\{(.*)\}\}/$1/);
+        push(@scheduled, parse_schedule_module($schedule, $module));
```
Comment (foursixnine) [inline lib/scheduler.pm:54]: Please add to the documentation: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/declarative-schedule-doc.md that now nested conditions are possible

Comment (SergioAtSUSE) [reply lib/scheduler.pm:54]: And also adapt `tools/detect_nonexistent_modules_in_yaml_schedule`

Comment (asdil12) [reply lib/scheduler.pm:54]: Done

**Review by rwx788 — COMMENTED** [review]
LGTM

**Review by jknphy — COMMENTED** [review]
I guess we will need some unit test as well in [t](https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/t)

**Review by foursixnine — APPROVED** [review]

**Review by SergioAtSUSE — APPROVED** [review]

**Pull request merged**